cfgov/jinja2/v1/_includes/molecules/related-posts.html

```diff
@@ -0,0 +1,109 @@
+{# ==========================================================================
+
+   related_posts.render()
+
+   ==========================================================================
+
+   Description:
+
+   Creates related posts markup when given:
+
+   settings:           An object used to customize the markup.
+
+   settings.post:      A post from a query result.
+
+   settings.posts:     An array of related posts.
+
+   settings.fullWidth: Boolean indication whether the related
+                       posts are full width.
+
+   ========================================================================== #}
+
+{% macro render(settings) %}
+{% from 'macros/util/format/url.html' import slugify as slugify %}
+{% set defaults = {
+    'search_types': 'posts, newsroom, press release speech, testimony',
+    'search_size':  5
+} %}
+{% do settings.update(defaults) %}
+{% set posts = settings.posts or
+               more_like_this(settings.post,
+                              search_types=settings.search_types,
+                              search_size=settings.search_size)
+%}
+{%  set category_lookup = {
+      'at the cfpb':              ('cf-icon-bullhorn', 'At the CFPB'),
+      'blog':                     ('cf-icon-speech-bubble', 'Blog'),
+      'data, research & reports': ('cf-icon-chart', 'Reports'),
+      'info for consumers':       ('cf-icon-information', 'Consumer Info'),
+      'newsroom':                 ('cf-icon-newspaper', 'Newsroom'),
+      'op-ed':                    ('cf-icon-contract', 'Op-Ed'),
+      'policy & compliance':      ('cf-icon-bank-account', 'Policy and Compliance'),
+      'press release':            ('cf-icon-bullhorn', 'Press Release'),
+      'speech':                   ('cf-icon-microphone', 'Speech'),
+      'testimony':                ('cf-icon-double-quote', 'Testimony')
+    }
+%}
+<div class="m-related-posts">
+    <h2 class="header-slug">
+        <span class="header-slug_inner">
+            Related Posts
+        </span>
+    </h2>
+    {#% TODO: Modify condition once related posts return various types %#}
+    {% if posts == false %}
+    {% for post in posts %}
+    {% set category = category_lookup[posts_section.category | lower] %}
+    <section class="m-related-posts_section">
+        <h4>
+            <span class=" cf-icon {{ category_icon[0] }}"></span>
+            {{ category[1] or 'Feed' }}
```

Comment:
Can we use a keyword index instead of `[1]`? It's not clear what the numerical index is looking up.